src/SQLChecks/Tests/SQLInstance.tests.ps1

```diff
@@ -5,13 +5,13 @@ Param(
 $serverInstance = $config.ServerInstance
 
 Describe "Trace flags" -Tag TraceFlags {
-    It "Correct global trace flags set on $serverInstance" {
-        @(Test-TraceFlags $Config).Count | Should Be 0
+    It "Correct global trace flags set on '$($serverInstance)'" {
+        (Test-TraceFlags $Config | ConvertTo-Json) | Should BeNullOrEmpty
```

Comment:
The output should be both human-readable and parsable. Compress isn't human-readable

I cannot drop the json formatting because that would break the test. I need to compare a single value against a NullOrEmpty

Reply:
Fair enough. Bear in mind though that for Pester v4 at least, `-BeNullOrEmpty` operates against arrays intuitively.

```powershell
ipmo Pester -MaximumVersion 4.99
$foo = Get-Process | Select -First 3 Id, ProcessName
Describe "foo" {
    It "Should exist" {
        $foo | Should -Not -BeNullOrEmpty
    }
}
```

![image](https://user-images.githubusercontent.com/10898767/232045642-120a6978-7dfa-45fb-84b5-0f60ae52a3fc.png)

Reply:
I was confused because the code sample didn't match the screenshot. The sample has an extra  `-Not`